<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainChatWin</class>
 <widget class="QWidget" name="MainChatWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>20</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>AvenirNext LT Pro Bold</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainChatWin</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/MainIco.ico</normaloff>:/MainIco.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame {
	background-color: rgb(245, 246, 247);
	border-width: 1px;
	border-radius: 10px;	
}

#closeBtn {
	border-radius: 10px;
	background-color: rgb(235, 59, 90);
}

#maxBtn {
	border-radius: 10px;
	background-color: #a5b1c2;
}

#minBtn {
	border-radius: 10px;
	background: #f7b731;
	
}

#sendBtn {
	border-radius: 8px;
	margin: 2px;
	color: rgb(255, 255, 255);
	background-color: rgb(30, 111, 255);
	font: 11pt &quot;AvenirNext LT Pro Bold&quot;;
}

#textArea {
	border: none;
	font: 14pt &quot;Leelawadee UI&quot;;
}

#chatMsgList {
	border-style: solid;
	border-color: #E0E0E0;
	border-top-width: 1px;
	border-bottom-width: 1px;
	border-left-width: 0;
	border-right-width: 0;
}

QListWidget::item:hover {
	background-color: #E8E8E8;

}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>1</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="rightMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>1</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>308</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <widget class="QPushButton" name="minBtn">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="images.qrc">
                <normaloff>:/images/min.svg</normaloff>:/images/min.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="maxBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="images.qrc">
                <normaloff>:/images/max.svg</normaloff>:/images/max.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="closeBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="images.qrc">
                <normaloff>:/images/close.svg</normaloff>:/images/close.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="4,22,1,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="styleSheet">
             <string notr="true">QWidget{
  background:white;
}</string>
            </property>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>10</y>
               <width>211</width>
               <height>50</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <widget class="QToolButton" name="uPortrait">
                <property name="styleSheet">
                 <string notr="true">QToolButton{
  background:transparent;
  border:none;
}</string>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="images.qrc">
                  <normaloff>:/images/img/otherside.png</normaloff>:/images/img/otherside.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="spacing">
                 <number>1</number>
                </property>
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>8</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="uName">
                  <property name="font">
                   <font>
                    <family>AvenirNext LT Pro Regular</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                    <stylestrategy>PreferDefault</stylestrategy>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel{
  background:transparent;

}</string>
                  </property>
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Arwen &lt;/span&gt;&lt;span style=&quot; color:#5b5b5b; vertical-align:sub;&quot;&gt;(0002)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="uQuote">
                  <property name="font">
                   <font>
                    <family>AvenirNext LT Pro Regular</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                    <stylestrategy>PreferDefault</stylestrategy>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel{
  background:transparent;
  color:gray;

}</string>
                  </property>
                  <property name="text">
                   <string>Hello Arwenleo!</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QPushButton" name="chathistoryBtn">
             <property name="geometry">
              <rect>
               <x>370</x>
               <y>30</y>
               <width>211</width>
               <height>31</height>
              </rect>
             </property>
             <property name="font">
              <font>
               <family>AvenirNext LT Pro Heavy</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton{
  background:transparent;
  color:#A0A0A0;
  border:none;
}

QPushButton:hover{
  background:transparent;
  color:#303030;
}</string>
             </property>
             <property name="text">
              <string>Get Chat History !</string>
             </property>
            </widget>
            <widget class="QWidget" name="layoutWidget">
             <property name="geometry">
              <rect>
               <x>594</x>
               <y>10</y>
               <width>161</width>
               <height>50</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <property name="spacing">
               <number>7</number>
              </property>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>1</number>
                </property>
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>8</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="mName">
                  <property name="font">
                   <font>
                    <family>AvenirNext LT Pro Regular</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                    <stylestrategy>PreferDefault</stylestrategy>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel{
  background:transparent;

}</string>
                  </property>
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Arwenleo &lt;/span&gt;&lt;span style=&quot; color:#5b5b5b; vertical-align:sub;&quot;&gt;(0001)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="mQuote">
                  <property name="font">
                   <font>
                    <family>AvenirNext LT Pro Regular</family>
                    <pointsize>12</pointsize>
                    <weight>50</weight>
                    <bold>false</bold>
                    <stylestrategy>PreferDefault</stylestrategy>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">QLabel{
  background:transparent;
  color:gray;

}</string>
                  </property>
                  <property name="text">
                   <string>Hello Arwen!</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QToolButton" name="mPortrait">
                <property name="styleSheet">
                 <string notr="true">QToolButton{
  background:transparent;
  border:none;
}</string>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="images.qrc">
                  <normaloff>:/images/img/myside.png</normaloff>:/images/img/myside.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>35</width>
                  <height>35</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="chatMsgList"/>
          </item>
          <item>
           <widget class="QTextEdit" name="textArea">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="sizeAdjustPolicy">
             <enum>QAbstractScrollArea::AdjustIgnored</enum>
            </property>
            <property name="undoRedoEnabled">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Write a message : )</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>1</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>388</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="sendBtn">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>30</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>AvenirNext LT Pro Bold</family>
                <pointsize>11</pointsize>
                <weight>50</weight>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="text">
               <string>SEND</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainChatWin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainChatWin</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>560</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>541</x>
     <y>57</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeWindowState()</slot>
  <slot>addMsgItem()</slot>
 </slots>
</ui>
